code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -896,7 +896,7 @@
 	assignedrole = "Lost Assistant"
 	important_info = "You are very much obsessed with the dragon. Do NOT stop thinking about the dragon."
 	outfit = /datum/outfit/job/assistant
-	mob_species = /datum/species/ipc
+	mob_species = /datum/species/ipc // Void edit
```

Comment:
this is gonna get deleted when we rebase so it might not be worth noting as a void edit so we know its gonna get chopped